web-client/src/components/RoleInfo/RoleInfo.tsx

```diff
@@ -0,0 +1,103 @@
+import { AlertTwoTone, HeartTwoTone } from '@ant-design/icons';
+import { Button, Col, Row } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+// import logo from '../../assets/logo.png';
+import cav from '../../assets/role_cav.png';
+import pin from '../../assets/role_pin.png';
+import TitleWithAddon from '../TitleWithAddon/TitleWithAddon';
+
+// const { Title, Text } = Typography;
+
+const StyledIntro = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 50px 50px;
+`;
+
+// const Logo = styled.img`
+//   height: 125px;
+//   width: 125px;
+// `;
+
+// const StyledTitle = styled(Title)`
+//   margin-top: 20px;
+//   margin-bottom: 50px !important;
+// `;
+
+// const Info = styled(Text)`
+//   margin-top: 40px;
+//   text-align: center;
+//   font-size: 20px;
+// `;
+
+// const Coll = styled(Col)`
+//   text-align: center;
+// `;
```

Comment:
Is this to use later or you just forgot to delete?

Reply:
Forgot to delete. Fixed.